java/arcs/core/testutil/BUILD

```diff
@@ -4,14 +4,30 @@ licenses(["notice"])
 
 package(default_visibility = ["//visibility:public"])
 
+HELPERS_SRC = ["HandleHelpers.kt"]
+
 # The `testutil` library can be JVM-only, since our unit tests are run on the JVM.
 arcs_kt_jvm_library(
     name = "testutil",
     testonly = True,
-    srcs = glob(["*.kt"]),
+    srcs = glob(
+        ["*.kt"],
+        exclude = HELPERS_SRC,
+    ),
     deps = [
         "//third_party/java/junit:junit-android",
         "//third_party/java/truth:truth-android",
         "//third_party/kotlin/kotlinx_coroutines",
     ],
 )
+
+arcs_kt_jvm_library(
+    name = "helpers",
+    testonly = True,
+    srcs = HELPERS_SRC,
+    deps = [
+        "//java/arcs/core/entity",
+        "//java/arcs/core/storage",
+        "//third_party/kotlin/kotlinx_coroutines",
+    ],
+)
```

Comment:
How come you've put this in a separate library?

Reply:
The android e2e and systemhealth tests don't build if they reference `//third_party/java/truth`.

Reply:
(Also, it'll probably turn out useful to be able to pull in the helper methods without dragging along unwanted test utils.)

Reply:
I'd suggest pulling this file out into a separate package then (e.g. `testutil.handles` or similar). Splitting a single folder into multiple different library targets gets messy fast.

Reply:
Seems a bit excessive for a pretty simple utility (plus we do have multi-target dirs elsewhere in the code).

Reply:
I regret doing that. It's annoying to unpick later on when it starts getting messy.

I think a package for test utils that you are able to use in e2e tests makes sense.

Reply:
Ok, done.